core/src/test/java/io/grpc/internal/ManagedChannelImplIdlenessTest.java

```diff
@@ -205,15 +204,14 @@ public void enterIdleModeAfterForceExit() throws Exception {
     newTransports.drainTo(transports);
     assertEquals(addressGroupList.size(), transports.size());
 
-    InterimTransport<ClientTransport> interimTransport = channel.tm.createInterimTransport();
+    channel.tm.createInterimTransport();
```

Comment:
I view tests as being examples/documentation how code should be used.  Since the real code would do it, it should be here.